### PR TITLE
[RU] Chapter 3 1/2/3 sections finished

### PR DESCRIPTION
**Pull request by pdumin**
Finished 1/2/3 sections of third chapter

### CODE REVIEW

File: chapters/ru/_toctree.yml [hunk 1]
```diff
@@ -30,3 +30,12 @@
...
     title: Введение
 
+
+- title: 3. Fine-tuning предобученной модели
+  sections:
+  - local: chapter3/1
+    title: Введение
+  - local: chapter3/2
+    title: Предобработка данных
+  – local: chapter3/3
```
Comment (lewtun) [inline chapters/ru/_toctree.yml:40]: We need to fix this character to build the content :)
```suggestion
  - local: chapter3/3
```

**Review by lewtun — APPROVED** [review]
Thank you for translating these sections @pdumin 🤗 ! It all looks good to me and there's just a small fix we need in the table of contents to build the website

**Comment by pdumin** [comment]
> Thank you for translating these sections @pdumin 🤗 ! It all looks good to me and there's just a small fix we need in the table of contents to build the website

I searched for a problem for a long time, I did not find it. Thank you, I'll be more careful

**Comment by lewtun** [comment]
Thanks for the fix, this looks great!

**Pull request merged**